htdocs/install/mysql/tables/llx_projet.sql

```diff
@@ -48,6 +48,10 @@ create table llx_projet
   usage_task           integer DEFAULT 1,		-- Set to 1 if project is used to manage tasks and/or record timesheet
   usage_bill_time      integer DEFAULT 0,		-- Set to 1 if time spent must be converted into invoices
   usage_organize_event integer DEFAULT 0,		-- Set to 1 if you want to use project to organize an event, receive attendees subscription				
+  allow_unknown_people_conf integer DEFAULT 0,		-- Set to 1 if you want to allow unknown people to suggest conferences
```

Comment:
Name of field is not clear. Can you rename into 
accept_conference_suggestions and
accept_booth_suggestions ?

Reply:
Done